apps/web/lib/connector.ts

```diff
@@ -22,9 +22,13 @@ const trezor = trezorModule({
   basePaths,
   assets,
 });
-const walletConnect = walletConnectModule();
+// const walletConnectV1 = walletConnectModule({ version: 1 });
```

Comment:
Normally I would say remove this but I can see you've got conditional logic in other locations for v1, so I think it's fine to leave here in case we need to revert/add logic to switch between them easily.

Reply:
yeah, since FE can't have v1 and v2 enabled simultaneously, it's intentionally commented and not removed so have an example on how to use v1.